main.go

```diff
@@ -17,7 +19,7 @@ func main() {
 	httpClient := &http.Client{
 		Timeout: time.Second * 3,
 	}
-	analyticsClient := &analytics.Client{HTTPClient: httpClient}
+	analyticsClient := &analytics.Client{HTTPClient: httpClient, ID: uuid.New().String()}
```

Comment:
Since a new analyticsClient is created for every new command run. We can just this to our advantage and associate a ID.

Reply:
Could we get rid of setting the ID in `CmdRunEventProperties`?

Reply:
Done